bitgo-java-api/setting.gradle

```diff
@@ -0,0 +1,4 @@
+rootProject.name = bitgo
+
+// For publishing ensure we will work with gradle 5.0 https://docs.gradle.org/4.9/userguide/publishing_maven.html#publishing_maven:deferred_configuration
+//enableFeaturePreview('STABLE_PUBLISHING')
```

Comment:
Does this need to be done?

Reply:
No, removed this comment.  This was from experimenting with publishing jars, but it's being handled by jitpack now.